src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -87,25 +99,59 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
     address: this.startingAddress,
     addressErrors: {},
     addressTouched: {},
-    phoneNumber:
-      this.props.order.requestedFulfillment &&
-      (this.props.order.requestedFulfillment.__typename === "CommerceShip" ||
-        this.props.order.requestedFulfillment.__typename === "CommercePickup")
-        ? this.props.order.requestedFulfillment.phoneNumber
-        : "",
+    phoneNumber: this.startingPhoneNumber,
     phoneNumberError: "",
     phoneNumberTouched: false,
+    showEditModal: false,
+  }
+
+  get startingPhoneNumber() {
+    const defaultPhoneNumber = this.defaultAddress?.phoneNumber
+    if (defaultPhoneNumber) {
+      return defaultPhoneNumber
+    } else {
+      return this.props.order.requestedFulfillment &&
+        (this.props.order.requestedFulfillment.__typename === "CommerceShip" ||
+          this.props.order.requestedFulfillment.__typename === "CommercePickup")
+        ? this.props.order.requestedFulfillment.phoneNumber
+        : ""
+    }
   }
 
   get startingAddress() {
-    return {
-      ...emptyAddress,
-      country: this.props.order.lineItems.edges[0].node.artwork.shippingCountry,
+    const defaultAddress = this.defaultAddress
```

Comment:
suggestion:
What do you think about renaming these to `defaultSavedAddress` or something similar to make it a bit more describtive?

Reply:
yeah i'm down for this! Great suggestion. Thank you for your comments and ALL YOUR HELP on this @sepans